src/data/members/fabien/experiences/darvaUI.js

```diff
@@ -9,29 +9,28 @@ module.exports = {
     color: 'white',
   },
   dates: {
-    from: Date.UTC(2019, 05),
+    from: Date.UTC(2020, 0),
     to: Date.now(),
   },
   informations: [
     { text: 'DARVA est une entreprise experte en solutions web et EDI (Echanges de Données Informatisés) basée à Niort et destiné à tous les acteurs professionnels de l\'assurance.' },
-    { text: 'Avec Guillaume CRESPEL, nous aidons le service BI à faciliter l\'accès et la gouvernance des "données sinistres" pour les collaborateurs de Darva ainsi que les acteurs des métiers de l\'assurance.' },
-    { text: 'Nous agissons sur le stockage des données des assureurs au travers de batch NodeJS, mais aussi sur l\'exploitation de celle-ci avec une UI utilisant l’écosystème ReactJS.' },
+    { text: 'Avec Guillaume CRESPEL, nous aidons le service BI à faciliter l\'accès et la gouvernance des "données sinistres" pour les collaborateurs de DARVA ainsi que les acteurs des métiers de l\'assurance.' },
+    { text: 'Nous simplifions la visualisation et l\'édition de la structure de données des assureurs grâce à une interface graphique ergonomique à base d\'arbre et de graphe en utilisant l’écosystème ReactJS.' },
```

Comment:
```suggestion
    { text: 'Nous simplifions la visualisation et l\'édition des structures de données assureurs grâce à une nouvelle interface graphique. Le but est entre autre de remplacer un veil outil d'édition, et de permettre à tous les collaborateurs de reprendre possession de la connaissance liée à ces données.' },
    { text: 'Pour faciliter l'accès à cette connaissance nous nous devons de proposer une interface graphique ergonomique, réactive, et compréhensible. Nous nous sommes appuyés sur des représentations graphiques bien connues comme des arbre (treeview) ou des graphes (radial). Et pour l'édition, quand cela semblait opportun, nous utilisons du drag'n'drop. Le tout en utilisant l’écosystème ReactJS.' },
```

(proposition, à revoir)